include/deal.II/sundials/arkode.h

```diff
@@ -975,6 +975,27 @@ namespace SUNDIALS
      */
     LinearSolveFunction<VectorType> solve_linearized_system;
 
+
+    /**
+     * Set the given @p solver for the solution of the linearized system.
+     *
+     * @note Alternative to setting solve_linearized_system directly.
+     */
+    template <typename SolverType>
+    void
+    set_linear_solver(SolverType &solver);
+
+    /**
+     * Set the given @p solver with SolverControl @p control for the solution of
+     * the linearized system. The @p control will receive tolerances directly
+     * from SUNDIALS.
+     *
+     * @note Alternative to setting solve_linearized_system directly.
+     */
+    template <typename SolverType>
+    void
+    set_linear_solver(SolverType &solver, SolverControl &control);
```

Comment:
Is there really no way to ask a solver for its SolverControl object?

Reply:
If there isn't, just write a patch for it in a separate PR. This certainly would make sense.

Reply:
TrilinosWrapper::SolverBase has such a function: https://www.dealii.org/developer/doxygen/deal.II/classTrilinosWrappers_1_1SolverBase.html#a271b791d8cb267ae211c56a0499385c4.